packages/contractkit/src/wrappers/Election.ts

```diff
@@ -100,6 +110,51 @@ export class ElectionWrapper extends BaseWrapper<Election> {
     this.contract.methods.getGroupsVotedForByAccount
   )
 
+  async getVotesForGroupByAccount(account: Address, group: Address): Promise<GroupVote> {
+    const pending = await this.contract.methods
+      .getPendingVotesForGroupByAccount(group, account)
+      .call()
+    const active = await this.contract.methods
+      .getActiveVotesForGroupByAccount(group, account)
+      .call()
+    return {
+      group,
+      pending: toBigNumber(pending),
+      active: toBigNumber(active),
+    }
+  }
+
+  async getVoter(account: Address): Promise<Voter> {
+    const groups = await this.contract.methods.getGroupsVotedForByAccount(account).call()
+    const votes = await Promise.all(groups.map((g) => this.getVotesForGroupByAccount(account, g)))
+    return { address: account, votes }
+  }
+
+  /**
+   * Returns whether or not the account has any pending votes.
+   * @param account The address of the account casting votes.
+   * @return The groups that `account` has voted for.
+   */
+  async hasPendingVotes(account: Address): Promise<boolean> {
+    const groups = await this.contract.methods.getGroupsVotedForByAccount(account).call()
+    const isNotPending = await Promise.all(
+      groups.map(async (g: string) =>
```

Comment:
Personal preference/Optional Nit: Type `const groups` to avoid having to type the argument here